filetests/isa/x86/simd-arithmetic-binemit.clif

```diff
@@ -120,6 +120,21 @@ ebb0:
 }
 
 
+function %imul_i64x2() {
+ebb0:
+    [-, %xmm1]    v0 = vconst.i64x2 [1 2]
+    [-, %xmm2]    v1 = vconst.i64x2 [2 2]
+    [-, %xmm1]    v2 = imul v0, v1 ; bin: 62 52 8d 08 40 ca
+    ; 62, mandatory EVEX prefix
+    ; 52 = 0101 0010, X and R' set (incorrectly I think) and mm set to OF38 (correctly)
+    ; 8d = 1000 1101, W set (correctly, vvvv set to 1 (correctly), bit 2 always set, pp set to 01 (correctly)
+    ; 08 = 0000 1000, V' set to 1 (incorrectly, I believe, due to packing of bits in RegUnit)
```

Comment:
I don't think I understand how `RegUnit` is being packed, I'm afraid. EVEX (and VEX) support larger numbers of registers so I attempted to extract the next bit to use in the bits that EVEX uses to extend the set of possible registers (e.g. `XRBR'V'`)--this seems to give the wrong answer. How would I tell Cranelift that EVEX has more registers available?

Reply:
I've figured this out... finally 😄.